website/core/Footer.js

```diff
@@ -71,7 +71,7 @@ class Footer extends React.Component {
           />
         </a>
         <section className="copyright">
-          Copyright &copy; {currentYear} Facebook Inc.
+          Copyright &copy; 2019 Facebook Inc.
```

Comment:
Good catch, I need to check what's going on, but this change should not be necessary. We are using [Docusaurus](https://docusaurus.io/) to generate our documentation. The expression `{currentYear}` is replaced by the corresponding year when the page is loaded locally:

![image](https://user-images.githubusercontent.com/18663703/62655643-6d9bb300-b930-11e9-987d-7f774dac9826.png)